media-gfx/maim/maim-5.7.4.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Commandline tool to take screenshots of the desktop"
+HOMEPAGE="https://github.com/naelstrof/maim"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/naelstrof/maim.git"
+else
+	SRC_URI="https://github.com/naelstrof/maim/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3"
```

Comment:
This seems wrong/outdated too, since upstream licenses.txt says:
```

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.
```
The source repository also seems to include at least one file licensed under MIT.